lib/services/jellyfin_api.dart

```diff
@@ -398,17 +438,18 @@ abstract class JellyfinApi extends ChopperService {
     request: JsonConverter.requestFactory,
   )
   @Post(path: "/Sessions/Logout", optionalBody: true)
-  Future<dynamic> logout();
+  Future<Response<dynamic>> logout();
 
-  static JellyfinApi create() {
+  static JellyfinApi create(bool inForeground) {
```

Comment:
What does this do? 🤔

Reply:
it disables the temp URL stuff used while logging in for the background copy of the api because we don't need it and we don't have a registered api helper to check.